cmd/permify/permify.go

```
@@ -54,4 +55,4 @@ func main() {
 	if err := root.Execute(); err != nil {
 		os.Exit(1)
 	}
-}
+}
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofumpt)